topologies/kne/cisco_ixia.textproto

```diff
@@ -32,13 +32,8 @@ nodes: {
     os: "ios-xr"
     config: {
         file: "cisco.config"
-        image: "8000e:latest"
```

Comment:
Why was the image removed? It's convenient for us to always have an image specified, even if that means we rewrite it on import.

Reply:
I did it based on the @alexmasi  comment on the kne pr, I am ok with reverting the change if u both agree.

Reply:
Yes Moji, for this repo we prefer to leave the image string explicitly in the proto (but thats not the case for the kne repo examples)

Sorry about any confusion there

Reply:
sure, done